tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Features/Organisations GetById 5147.feature

```diff
@@ -16,6 +16,16 @@ Scenario: 1. Get the details of a single organisation
 	Then a response with status code 200 is returned
 	And the string value of name is Organisation 1
 	And the string value of odsCode is Ods 1
+	And the string value of primaryRoleId is ID 1
```

Comment:
I compared this section with the GET all organisation tests & I wondered why the checking of the result is different. Do you think it would be a good idea (if possible) to utilize the approached used in the get all feature file?